cli_test/helpers/test_helpers.go

```diff
@@ -0,0 +1,165 @@
+package helpers
```

Comment:
Could we rename this file into `helpers.go` please?

Reply:
Sure